src/services/encryptionService.js

```diff
@@ -37,7 +39,7 @@ class EncryptionService {
   async clearStoredAccount() {
     this.#account = null;
     this.#encryptionSessions = {};
-    await localforage.clear();
+    await garbageCleaningService.clearAllLocalData();
```

Comment:
this looks like a cycle, e.g. GC service calls EncryptionService which calls GC service etc..